CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 Changes
 =======
 
+# 4.0.0 / 2019-12-19
+
+## Breaking changes
+
+  * **This cookbook will install Agent 7.x by default**. Datadog Agent 7 uses Python 3 by
+  default so if you were running any custom checks written in Python, they must now be
```

Comment:
I'd remove the "by default", Agent 7 has no way to run Python 2 even with non-default config.

Reply:
👍